api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -65,12 +64,18 @@ message PluginConfig {
   // Filter/service configuration used to configure or reconfigure a plugin
   // (proxy_on_configuration).
   google.protobuf.Any configuration = 4;
+
+  // If there is a fatal error on the VM (e.g. exception, abort(), on_start or on_configure return false),
+  // then all plugins associated with the VM will either fail closed (by default), e.g. by returning an HTTP 503 error,
+  // or fail open (if 'fail_open' is set to true) by bypassing the filter. Note: when on_start or on_configure return false
```

Comment:
`return false`

Reply:
It says "return false", but "false" is outside of the default viewing window, so you need to scroll to the right to see it.

Reply:
I am not sure what this comment is referring to, could you clarify?

Reply:
Sorry, I was working in a window without Piotr's comments.